packages/js/components/package.json

```diff
@@ -44,6 +44,7 @@
 		"@woocommerce/navigation": "workspace:*",
 		"@wordpress/a11y": "3.5.0",
 		"@wordpress/api-fetch": "^6.0.1",
+		"@wordpress/base-styles": "^4.3.0",
```

Comment:
The lockfile wasn't updated when I did `pnpm add @wordpress/base-styles@^4.3.0` to add this. Should it be?

Reply:
https://github.com/ROBERLANDIO/-octaedro-approved-this-pull-request.-Nice-job-joshuatf-This-is-testing-well-and-the-code-